ts/features/itwallet/onboarding/screens/WalletCardOnboardingScreen.tsx

```diff
@@ -42,19 +45,28 @@ import {
 import { ItwCredentialIssuanceMachineContext } from "../../machine/provider";
 import { ItwOnboardingModuleCredential } from "../components/ItwOnboardingModuleCredential";
 import { useOfflineToastGuard } from "../../../../hooks/useOfflineToastGuard.ts";
+import { ITW_ROUTES } from "../../navigation/routes";
+import { selectItwEnv } from "../../common/store/selectors/environment";
 
-// List of available credentials to show to the user
+// Credentials that can be actively requested and obtained by the user
 const availableCredentials = [
   CredentialType.DRIVING_LICENSE,
   CredentialType.EUROPEAN_DISABILITY_CARD,
   CredentialType.EUROPEAN_HEALTH_INSURANCE_CARD
 ] as const;
 
+// Credentials that will be available in the future
+const upcomingCredentials = [CredentialType.DEGREE_CERTIFICATES] as const;
+
 const activeBadge: Badge = {
   variant: "success",
   text: I18n.t("features.wallet.onboarding.badge.active")
 };
 
+// Helper function to check if a credential type is upcoming
+const isUpcomingCredential = (type: CredentialType): boolean =>
+  (upcomingCredentials as readonly CredentialType[]).includes(type);
```

Comment:
When referring to credential types is better to use a simple `string`
```suggestion
const isUpcomingCredential = (type: string): boolean =>
  upcomingCredentials.includes(type as any);
```

Reply:
Thanks for the suggestion! Addressed in [9354106](https://github.com/pagopa/io-app/pull/7100/commits/9354106944a25624ca15e153625721a0a8bffe45)